docs/book/src/topics/ignition-support.md

```diff
@@ -0,0 +1,127 @@
+# Ignition support
```

Comment:
@invidian - one last thought on the docs. Do you think it would be worth calling out this is experimental and the feature flag name at the top like we do for [machine pools](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/docs/book/src/topics/machinepools.md)?

Reply:
Make sense. Done, thanks!